Shared/Objects/PosterType.swift

```diff
@@ -40,7 +40,7 @@ enum PosterType: String, CaseIterable, Displayable, Defaults.Serializable {
 
         static let landscape = 350.0
         #else
-        static var portrait = 100.0
+        static var portrait = 120.0
```

Comment:
This changes the portrait poster size _everywhere_ and we don't want that. Instead, you'll have to do some logic with the poster type and scale with the `scaleItem` modifier:

https://github.com/jellyfin/Swiftfin/blob/18b06ff2972ad61d3da34cb112d0eceaf301d91c/Swiftfin/Components/PagingLibraryView.swift#L65

Reply:
Reworked to use scaling with the new commit!